packages/lab/src/combo-box-next/ComboBoxNext.css

```diff
@@ -0,0 +1,13 @@
+.saltComboBoxNext-highlight {
+  font-weight: var(--salt-text-fontWeight-strong);
+}
+
+.saltComboBoxNext-input:hover {
+  --input-borderColor: var(--salt-focused-outlineColor);
+}
+
+.saltComboBoxNext-list {
+  --comboBoxNext-itemCount: 5;
```

Comment:
where/when does this get redefined? I can't see anywhere atm so dont think it needs to be a variable